cross/znc/modules/clientbuffer.cpp

```diff
@@ -0,0 +1,626 @@
+/*

```

Comment:
Please provide (in this header comment for instance) information about source code origin (GitHub URL or any other location) and current version of this copied file so that maintainer is able to track and apply updates

Reply:
Ok this is done.

Reply:
I would like to see this in `spk/znc/Makefile` to keep it consistent with the other modules.

Please add this line to spk/znc/Makefile  in the `znc_post_install` target below the other modules:
```
  # wget https://raw.githubusercontent.com/CyberShadow/znc-clientbuffer/master/clientbuffer.cpp
```

and reset `cross/znc/modules/clientbuffer.cpp` to its original content.

This makes the update of the modules as easy as to build with (temp.) uncommented wget commands in the post installation target.